eZ/Publish/Core/Search/Solr/Content/Gateway/EndpointProvider/TranslationEndpointProvider.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/**
+ * This file is part of the eZ Publish Kernel package
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+
+namespace eZ\Publish\Core\Search\Solr\Content\Gateway\EndpointProvider;
+
+use eZ\Publish\Core\Search\Solr\Content\Gateway\EndpointProvider;
+use RuntimeException;
+
+/**
+ *
+ */
+class TranslationEndpointProvider implements EndpointProvider
+{
+    /**
+     * @var array
+     */
+    protected $endpointMap;
+
+    public function __construct()
+    {
+        $this->endpointMap = array(
+            "eng-GB" => "http://localhost:8983/solr/core0",
```

Comment:
could we have a mapping that consists of repository(or db) name + language name or similar? (to make sure multi repository is possible)